### PR TITLE
Update readme #24

### PR DESCRIPTION
**Pull request by Vaishnavi1047**
## Description
Update readme so that steps are provided to create api key while connecting with omdb api.

## Screenshots (if appropriate):
Screenshot 2024-10-18 001755

## Checklist:

- [x] I have registered myself at [Contrihub website](https://sac.mnnit.ac.in/contrihub).
- [x] My code follows the code style of this project.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

**Comment by Vaishnavi1047** [comment]
@eleensmathew have a look at it mam and tell me is there any need to update?

File: README.md [hunk 1]
```diff
@@ -76,6 +76,36 @@ Before coding, you need to install Django in your virtual environment:
...
+1. Create a .env file in the root directory of your project:
+```bash
+touch .env
+```
+2. Add your OMDB API key to the .env file:
+```bash
+OMDB_API_KEY=your_api_key_here
+```
+3. In your Django settings or configuration file, load the API key from the .env file:
+ ```bash
```
Comment (eleensmathew) [inline README.md]: This is already present in code. pls remove this step.

Comment (Vaishnavi1047) [reply README.md]: okay mam

Comment (Vaishnavi1047) [reply README.md]: Screenshot 2024-10-18 155624
is this fine now?

Comment (eleensmathew) [reply README.md]: We have defined the view code as API_KEY = os.getenv('API_KEY') so the name has to be chaged.

Comment (Vaishnavi1047) [reply README.md]: Screenshot 2024-10-18 161625

Comment (eleensmathew) [reply README.md]: Since in the code we have used API_KEY instaed of OMDB_API_KEY. Can you change it to API_KEY = secret_key?

Comment (Vaishnavi1047) [reply README.md]: Screenshot 2024-10-18 165532

Comment (eleensmathew) [reply README.md]: LGTM

Comment (Vaishnavi1047) [reply README.md]: may i make pull request now?

Comment (eleensmathew) [reply README.md]: Sure but also do a pull request so that there arent merge conflicts.

Comment (Vaishnavi1047) [reply README.md]: @eleensmathew i tried but same pr me merge ho rha h

**Pull request merged**